script.js

```diff
@@ -1,4 +1,129 @@
-var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
-};
+//2 players and each take tuns
+//When a player clicks Submit, the game rolls 2 dice and shows the dice rolls
+//The player picks the order of the dice they want
+//choose how the player specifies dice order
+//After both players have rolled and chosen dice order, the player with the higher combined number wins.
+
+var start = "Start of game";
+var chooseDiceOrder = "Choose Dice Order";
+var compareFinalNumb = "Compare Final Number";
+var gameMode = start;
+
+//Determine player
+var startingPlayer = 1;
+
+//Dice tracking
+var player1Dice = [];
+var player2Dice = [];
+
+//Track choice
+var player1choice;
+var player2choice;
+
+function main(input) {
+  //Start game, roll dice for 1st player
+  if (gameMode == start) {
+    var startingRoll = getDice();
+    gameMode = chooseDiceOrder;
+    return `Hi! Player ${startingPlayer}. <br>
+    Your first Dice is ${startingRoll[0]} and your second Dice is ${startingRoll[1]} <br>
+    Please choose your first digit base on your dice, 1 or 2`;
+  }
+
+  //Move onto chosing dice
+  if (gameMode == chooseDiceOrder) {
+    var diceChoice = Number(input);
+    //Validate input
+    if (diceChoice !== 1 && diceChoice !== 2) {
+      return "Please select 1st or 2nd dice to proceed";
+    }
+
+    //Combine the 2 dice base on dice choice
+    var combinedNumb = getPlayerChoice(diceChoice);
```

Comment:
Notice how for your `getPlayerChoice` function, you have to check if starting player is 1 or 2. But you're already doing this check on line 47 and 52. Why not we make use of the check on line 47 and 52 so that our helper function does not need to do the check. What that would look like is:
inside line 47, we call the `getPlayerChoice` function as follows `getPlayerChoice(diceChoice, player2Dice)`
and inside line 52 `getPlayerChoice(diceChoice, player1Dice)`

try this out and see if it makes sense to you. If not, feel free to reach out and we can talk about it(: